models/public/open-closed-eye-0001/description/open-closed-eye-0001.md

```diff
@@ -0,0 +1,34 @@
+# Model name: action-recognition-0001-decoder
+
+## Use Case and High-Level Description
+
+Fully convolutional network for recognition of eye state ('open', 'closed').
+
+## Example
+
+![](./1.png)
+![](./2.png)
+![](./3.png)
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Source framework                | PyTorch*                                  |
+| GFlops                          | 0.001                                     |
+| MParams                         | 0.01125                                   |
+
+
+## Performance
+
+## Inputs
```

Comment:
As it distributed as public model, it should has description for both original and converted models. Escpecially, if it has some mean-scale preprocessing. Also I ask you to recheck channel order for both models